apps/moose-console/sentry.server.config.ts

```diff
@@ -0,0 +1,15 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://ad3e8eb464c321b18b78c8402e80a8e5@o4505851966128128.ingest.sentry.io/4505852583084032",
```

Comment:
Did you update those values for with a new entry for the console in sentry?

Reply:
No :( )